build_tools/travis/install.sh

```diff
@@ -82,6 +82,10 @@ if [[ "$COVERAGE" == "true" ]]; then
     pip install coverage codecov
 fi
 
+if [[ "$PILLOW" == "true" ]]; then
```

Comment:
Could you merge master because your install.sh is outdated: https://github.com/scikit-learn/scikit-learn/blob/master/build_tools/travis/install.sh#L40

I would probably add it to the `TO_INSTALL` environment variable if possible

Reply:
I rebased on the current master and added the environment variable to `TO_INSTALL` for `"$DISTRIB" == "conda"`. The disadvantage is that I had to separately add it for the `"$DISTRIB" == "ubuntu"` case. Should I keep it like this?